aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -35,6 +35,13 @@ inline int dataSize(cudnnDataType_t dataType)
 // function modifies 'stride' in place so this invariant holds.
 static inline void fixSizeOneDimStride(int dim, const int *size, int *stride) {
   int64_t z = 1;
+  if (dim == 4) {
```

Comment:
Is this function used anywhere? Logic seems fishy to me - if nhwc-contiguous with already good 1-dim strides, bail out, but if say these strides are messed up, they are going to get messed up even more below.

Reply:
Good catch. Forgot to update this guy after the last commit that fixed `suggest_memory_format()`. This function is somewhat busted for NHWC. But we actually always have good strides for NHWC before entering a convolution, so it's not relevant.

I'll update it.

Reply:
I mean, I'll properly fix this function so we would properly fix SizeOneDim for NHWC as well.